### ISSUE TITLE
Terraform Azure how to use -auto-approve -input=false using terratest

### ISSUE THREAD

**Issue by ravigaddammto** [gruntwork-io/terratest#570]
Hello, Thank you for the great work at terratest.

I am using terratest to test Azure deployments.

This is the command I have to apply the terraform and I am trying to achieve the same thing from terratest
```
terraform apply -no-color -auto-approve -input=false -var-file "../environments/env/shared.tfvars" -var-file "../environments/env/cont_reg.tfvars" 
```

And here is my code from sample_test.go:
```
	terraformOptions := &terraform.Options{
		TerraformDir: "../scripts",
		NoColor: true,
		BackendConfig: map[string]interface{}{
			"storage_account_name": storageactname,
			"key": fname,
		},
```
How can I achieve the **-auto-approve** and **-input=false** part from the terratest apply?
Note: I am able to use ```-no-color``` with ```NoColor: true``` but not sure how to use ```-auto-approve``` and ```-input=false```

**Comment by tsalright** [comment]
@brikis98, I think this question is related to the core Terratest functionality of passing the `-auto-approve` and `-input=false` arguments to the `apply`. 

@ravigaddammto it looks like the `-auto-approve` and `-input=false` always get applied when you run any of the `Apply` functions. See code here: https://github.com/gruntwork-io/terratest/blob/2b91e96ce579ab52a6d120ad958928253c91de69/modules/terraform/apply.go#L53

**Comment by ravigaddammto** [comment]
@tsalright Yes, didn’t get a chance updating the issue here. These options are passed even without giving them exclusively.

Up to terratest team if they want to track this for any more options related to this. For my initial issue I think this can be closed.

**Comment by james03160927** [comment]
Closing this based on remark above.
